web/packages/shared/config.js

```diff
@@ -20,6 +20,8 @@ const cfg = {
   dateTimeFormat: 'yyyy-MM-dd HH:mm:ss',
   dateFormat: 'yyyy-MM-dd',
   shortFormat: 'MMM dd, yyyy',
+  // Displays time as 12/25/24 at 12:00AM
+  dateWithPrefixedTime: `LL/dd/yy 'at' h:mma`,
```

Comment:
Should date inside it be the same format as `dateFormat` above? "12/25/24" is ambiguous format for days <= 12 between US/Europe

Reply:
extended `yy` to `yyyy`

Reply:
I think Anton meant yyyy-MM-dd in general. 😏 But I know that the current format is just a temporary stopgap.